composer.json

```diff
@@ -16,9 +16,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^7.5.19",
-        "roave/security-advisories": "dev-master"
-    },
-    "suggest": {
```

Comment:
I think we'll still want to `suggest` these packages, even though we include them in `require-dev`

Reply:
Put back in both the require-dev and suggest sections.